### DISCUSSION TITLE
Spiegazione binding

### DISCUSSION THREAD

**Discussion by ster1um** [indomus/forum#2105] [ZigBee, Z-Wave, Thread, Matter]
Se non ho mal interpretato il bind tra dispositivi può fare a meno del coordinator e quindi funzionare anche in assenza di quest'ultimo.

Dovrei abbinare una presa con radiocomando ikea, è possibile abbinarli così da poter azionare la presa con il solo radiocomando.

**Comment by snakuzzo** [comment]
No...non è così.
il bind tra dispositivi NON può fare a meno del coordinator, visto che è quest'ultimo a "creare" e "gestire" la rete.

[QUI](https://www.zigbee2mqtt.io/guide/usage/binding.html#binding) lo trovi spiegato bene (proprio all'inizio)

**Reply by ster1um** [reply]
quindi a cosa serve fondamentalmente?

**Reply by snakuzzo** [reply]
Hai letto ? Chiedo preché se lo domandi probabilmente non hai nemmeno cliccato sul link :)
E' tutto scritto.

Ti faccio un esempio...
Ammettiamo tu voglia accendere una luce (pilotata da un attuatore zigbee) alla accensione di un'altra luce (pilotata da altro attuatore zigbee).
Senza il bind tra i due attuatori, dovresti fare un'automazione lato HA per poter realizzare questa cosa.
Con il bind di fatto non ti serve HA...in pratica ti basta tenere "alimentato" il coordinator. Non ti serve nemmeno che sia up&running zigbee2mqtt

**Reply by snakuzzo** [reply]
In realtà il bind ha bisogno del coordinator ( e del software) per essere impostato, ma dopo averlo impostato potresti anche escluderlo. Questo però dipende anche dal tipo di device. Quindi ti converrebbe fare delle prove direttamente "sul campo"

**Reply by snakuzzo** [reply]
Tempo fa avevo preso una presa IKEA con telecomandino a parte e di base il bind si poteva fare tra i due device senza la necessità del coordinator

**Reply by ster1um** [reply]
> Hai letto ? Chiedo preché se lo domandi probabilmente non hai nemmeno cliccato sul link :) E' tutto scritto.

l'ho letto ma ci ho capito poco... 

ho provato a fare un binding ma (magari per mia incapacità) non funziona... pazienza

Grazie mille comunque della spiegazione
